packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -159,69 +156,106 @@ const ProgressIndicatorItem = ({
 				},
 			}}
 		>
-			<Flex
-				as={as}
-				aria-current={isActive || undefined}
-				gap={1}
+			<Box
+				as="span"
+				aria-current={isActive || undefined} // TODO: Should this be 'step'? Also, I think this has some browser/screen-reader combo issues e.g. edge + NVDA and chrome/firefox/edge + JAWS (from A11y Slack)
```

Comment:
Remove the comment?

Reply:
Yep, these are both gone:

[level one item](https://github.com/agriculturegovau/agds-next/blob/5d20d6ed9eec652b34c5b56132429f0d67020211/packages/react/src/progress-indicator/ProgressIndicatorItem.tsx#L126)

[level two item](https://github.com/agriculturegovau/agds-next/blob/5d20d6ed9eec652b34c5b56132429f0d67020211/packages/react/src/progress-indicator/ProgressIndicatorItem.tsx#L85)